_component_design/chips.md

```diff
@@ -17,27 +17,87 @@ helix-ui-javascript: false
 
 <section class="static-section" markdown="1">
 
-## Introduction
+<div class="hxRow" markdown="1">
+
+{% column left:"hxCol-12 hxCol-xs-12 hxCol-sm-12 hxCol-md-12 hxCol-lg-12" %}
+
+## When to use
+
+Use chips to graphically represent a unit of information. For example, use chips to represent a user or device.
+
+## Best practices
+
+Consider the following best practices when constructing a chip:
+
+- Avoid using chips in multiple locations on a page.
+- Label the chip so that the user can identify the information represented by the chip.
+
+{% endcolumn %}
+
+
+</div>
+
+</section>
+
+<section class="static-section" markdown="1">
 
 <div class="hxRow" markdown="1">
 
 {% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
```

Comment:
Why is this in it's own col4 without an image next to it? If it's supposed to be next to the same image as the one next to the **Specifications** section, remove lines 52-62 as the content could all be in the same column. If it's not supposed to be next to the same image, then this should be a col8 per the discussion today in Slack. We don't have col4 for text unless there's an image next to it per sitewide style.

Reply:
I was only changing the spacing of when to use and best practices. But it probably makes sense to have all text not associated with an image be in a col8.